### DISCUSSION TITLE
Aiuti per principiante. Gruppi tapparelle e Utility meter

### DISCUSSION THREAD

**Discussion by lucaxxaa** [indomus/forum#292] [HUB: Home Assistant]
Ciao, due semplici quesiti, per me non troppo semplici. 

1- Per creare due gruppi tapparelle, zona notte e giorno, basta aggiungere le entita'  sensor degli shelly2.5,  al group..yaml , come per le luci ? e posso comandare separatamente i due gruppi ?  

2-con utility meter, come posso usare il dato last period ( dati del giorno, mese, anno precedente) a livello di sintassi ?

![image](https://user-images.githubusercontent.com/79480865/117613559-60e06a80-b167-11eb-8615-966191cce541.png)

**Comment by snakuzzo** [comment]
1. No...nel `groups.yaml` ad esempio crei...

Es.

```
zona_giorno:
  - cover.soggiorno
  - cover.cucina
  - cover.bagno_giorno
 
zona_notte:
  - cover.camera
  - cover.cameretta
  - cover.bagno_notte
```

2. Non ho capito la domanda

**Reply by Scialla** [reply]
Presumo chieda aiuto per il template.... devi usare un template per estrarre l'attributo

**Reply by lucaxxaa** [reply]
Si. Grazie per punto 1 avevo intuito

Per il punto 2, esatto devo capire come estrarre attribuito

**Reply by lucaxxaa** [reply]
```- platform: template
  sensors:
    door_sensor_battery_level:
      friendly_name: "Door Sensor Battery Level"
      entity_id: binary_sensor.openclose_13
      unit_of_measurement: '%'
      value_template: "{{ state_attr('binary_sensor.openclose_13', 'battery_level') }}"
```

**Reply by lucaxxaa** [reply]
Tipo cosi

**Reply by snakuzzo** [reply]
sì

P.S. Ti ho corretto il markdown
